src/bun.js/VirtualMachine.zig

```diff
@@ -2689,6 +2689,8 @@ pub fn remapZigException(
                 frames[j] = frame;
                 j += 1;
             }
+
+            @memset(frames[j..], jsc.ZigStackFrame.Zero);
```

Comment:
why is the correct fix?  remember that zig doesn't have automatic constructor/destructors, so unless we are calling `deref` on frames outside the bounds of `.len`, this change should not do anything. and if that is what's happening, then it should be fixed at those call sites and not at this call site.